sympy/concrete/expr_with_intlimits.py

```diff
@@ -113,18 +116,18 @@ def change_index(self, var, trafo, newvar=None):
                 beta = p.coeff_monomial(S.One)
                 if alpha.is_number:
                     if alpha == S.One:
-                        limits.append((newvar, alpha*limit[1] + beta, alpha*limit[2] + beta))
+                        limits.append((newvar, alpha * limit[1] + beta, alpha * limit[2] + beta))
```

Comment:
Here not having blanks around `*` is actually okay, it's indicating operator precedence.
(Also, PEP8 gives `hypot2 = x*x + y*y` as an example for okay code.)